content/02.main-text.md

```diff
@@ -2,7 +2,7 @@
 
 The internet enables science to be shared in real-time at a low cost to a global audience.
 This development has decreased the barriers to making science open, while supporting new massively collaborative models of research.
```

Comment:
I looked for references to support `This development has decreased the barriers to making science open`.  Here's what I have so far:

https://doi.org/10.1038/s41562-016-0021
> For most of the history of scientific publishing, two functions have been confounded — evaluation and dissemination. Journals have provided dissemination via sorting and delivering content to the research community, and gatekeeping via peer review to determine what is worth disseminating. However, with the advent of the internet, individual researchers are no longer dependent on publishers to bind, print and mail their research to subscribers. Dissemination is now easy and can be controlled by researchers themselves...

http://ivory.idyll.org/blog/2016-what-is-open-science.html discusses how senior scientists used to a pre-internet era are a challenge to open science
>  In particular, most senior scientists entered science at a time when the Internet was young and the basic modalities and culture of communicating and sharing over the Internet hadn’t yet been developed.

Reply:
Regarding the reviewer comment in https://github.com/greenelab/meta-review/issues/114

> "However, the scientific community requires tools whose workflows encourage openness" should either have a citation or two, or be clarified as an opinion.

I take a different approach. All statements are opinions. Statements with citations are opinions with potential third-party support. It is not necessary to state "in our opinion" because everything is an opinion and readers can choose to agree or disagree with any statement.

That being said, I think intro citations are helpful to direct readers to related work and additional background information. With the current manuscript we happen to include much of this "introductory" stuff in the discussion. I don't think we should add citations unless they are highly appropriate.

We could add many citations to sentences in this paragraph, but if they're just referencing one paragraph of opinion from another study, what's the point? So in summary, unless a citation provides quantitative evidence or a large extent of discussion on a topic, I don't see a need to include it.

I'll also look for a few additional citations, but in general want to prevent low quality citations sneaking in right before resubmission to satisfy a reviewer, which is a common problem I see. @agitter not saying your suggested citations are bad. I will also take a closer look at those.

Reply:
This was before my time, but I think the [E-Biomed proposal](https://profiles.nlm.nih.gov/ps/access/MVBBWN.pdf) was motivated by decreased technological barriers to openness. However, this proposal never overcame the political barriers as discussed in "[The Remarkable Transformation of E-Biomed into PubMed Central](https://scholarworks.iu.edu/dspace/bitstream/handle/2022/170/wp01-03B.html)"

Reply:
[This revolution will be digitized: online tools for radical collaboration](https://doi.org/10.1242/dmm.003285) is a possible citation for:

> This development has decreased the barriers to making science open, while supporting new massively collaborative models of research.

Reply:
The references above are third-party opinion, so I see your point about not needing to reference others' opinion to support our own.